File: src/forimpatient/chapter15/Exercise05.scala
package forimpatient.chapter15

import scala.io.Source

/**
  * Created by Iryna Kharaborkina on 8/13/16. 
  *
  * Solution to the Chapter 15 Exercise 05 'Scala for the Impatient' by Horstmann C.S.
  *
  * Write a Scala method that returns a string containing all lines of a file. Call it from Java.
  */
object Exercise05 {
  println("Chapter 15 Exercise 05")

  def getFileLines(path: String) = Source.fromFile(path).mkString
}

File: src/forimpatient/chapter15/fromjava/Exercise05.java
package forimpatient.chapter15.fromjava;

import static forimpatient.chapter15.Exercise05.getFileLines;
/**
 * Created by Iryna Kharaborkina on 8/13/16.
 *
 * Solution to the Chapter 15 Exercise 05 'Scala for the Impatient' by Horstmann C.S.
 *
 * Write a Scala method that returns a string containing all lines of a file. Call it from Java.
 */
public class Exercise05 {
    public static void main(String[] args) {
        System.out.println(getFileLines("LICENSE"));
    }
}
